src/main/resources/assets/js/application/report/services/tdprReportService.js

```diff
@@ -2,29 +2,22 @@ define(['application/report/tdprReportModule'], function (tdprReportModule) {
     tdprReportModule.service('tdprReportService', ['$http', '$q', 'dateFilter', function ($http, $q, dateFilter) {
         var format = 'dd-MM-yyyy';
 
-        this.getReport = function (dateFrom, dateTo, personId) {
-            return $http.get('/api/report/' + dateFilter(dateFrom, format) + '/' + dateFilter(dateTo, format) + '/' + personId)
-                .then(
-                    function (response) {
-                        return response.data;
-                    },
-                    function (error) {
-                        error.message = "Unable to get data from server!";
-                        return $q.reject(error.message)
-                    })
-        };
-
         this.getReports = function (dateFrom, dateTo) {
             return $http.get('/api/report/' + dateFilter(dateFrom, format) + '/' + dateFilter(dateTo, format))
                 .then(
                     function (response) {
-                        return response.data;
+                         var reports = response.data.map(sumOfHours);
+                         return reports;
```

Comment:
Not sure if it is necessary to assign to variable which is returned on next line. Maybe just do return response.data.map(sumOfHours);